DependencyInjection/Configuration.php

```diff
@@ -73,6 +73,14 @@ public function getConfigTreeBuilder()
 
                 ->scalarNode('title')->defaultValue('Sonata Admin')->cannotBeEmpty()->end()
                 ->scalarNode('title_logo')->defaultValue('bundles/sonataadmin/logo_title.png')->cannotBeEmpty()->end()
+
+                ->arrayNode('global_search')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->scalarNode('empty_boxes')->defaultValue('show')->end()
```

Comment:
what options are possible here? they should be validated, imo its `show`, `fade` and `hide`